docs/json/sonarr/rp/lqGroups.json

```diff
@@ -6,7 +6,7 @@
   "preferred": [{
     "score": -10000,
     "terms": [
-      "/(-BRiNK|-CHX|-CTFOH|-EVO|-FGT|-GHOSTS|-HiQVE|-iNTENSO|JFF|MeGusta|-NERO|nhanc3|Pahe\\.ph|Pahe\\.in|PSA|TBS|-TG|-VIDEOHOLE|-worldmkv|-XLF|-Zero00)\b/i"
+      "/(-BRiNK|-CHX|-CTFOH|-EVO|-FGT|-GHOSTS|-HiQVE|-iNTENSO|JFF|MeGusta|-NERO|nhanc3|Pahe\\.ph|Pahe\\.in|Pahe|PSA|TBS|-TG|-VIDEOHOLE|-worldmkv|-XLF|-Zero00)\b/i"
```

Comment:
can prob drop the Pahe.ph / Pahe.in since Pahe would take care of both?

Reply:
that might be the case yeah, been a long time since I was using RPs 😄